buddyboss/buddyboss-platform
# PROD-896 Thumbnail preview is not generated issue - fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1780  .

### How to test the changes in this Pull Request:

1. Make sure the Upload Size larger then 10 mb from BuddyBoss settings
2. Go to activity page and upload an image larger then 10 mb
3. See the thumbnail

### Proof Screenshots or Video

https://www.loom.com/share/1b5f1fb5bc044d9b93ecfe193c24631b

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
